### ISSUE TITLE
Numbers not showing 

### ISSUE THREAD

**Issue by ARRed98** [HaveAGitGat/Tdarr#922]
After updating tdarr i got this bug, i don't know why or how to reproduce it
![Screenshot 2023-12-03 091103](https://github.com/HaveAGitGat/Tdarr/assets/102137508/3db0fc4f-beee-4e1a-8cec-55d6b54d3c87)

**Comment by HaveAGitGat** [comment]
Hi, are there any files in the queues? Can see from the message that the transcode queue is empty.

**Comment by ARRed98** [comment]
yes there's files in the queue

**Comment by HaveAGitGat** [comment]
Have you tried restarting Tdarr or hard refreshing the browser to see if that corrects it? There were some slight changes to that in last version so could be that browser has cached and is still using previous web UI.

**Comment by ARRed98** [comment]
That's first thing i tried, i refreshed the page and restarted tdarr node and server and the bug seems to be permanent, I even rebooted the pc and still the same problem

**Comment by ARRed98** [comment]
I tried, it didn't help

**Comment by HaveAGitGat** [comment]
Are you able to go into the server data folder into `StatisticsJSONDB` and see how many files there are. There should only be one, are you able to past the contents thanks:
![image](https://github.com/HaveAGitGat/Tdarr/assets/43864057/5777fe56-0918-4744-9f5f-c0c8217fd08d)

**Comment by ARRed98** [comment]
{"_id":"statistics","totalFileCount":21633,"totalTranscodeCount":5837,"totalHealthCheckCount":100740,"sizeDiff":2916.681148480624,"DBFetchTime":"1s","DBLoadStatus":"Stable","DBQueue":0,"pies":[["All","all",21633,5837,2916.681148480624,100740,[{"name":"Not required","value":5742},{"name":"Queued","value":11004},{"name":"Transcode success","value":4708},{"name":"Transcode error","value":171},{"name":"Ignored","value":8}],[{"name":"Success","value":21540},{"name":"Queued","value":87},{"name":"Error","value":6}],[{"name":"hevc","value":16366},{"name":"h264","value":4788},{"name":"mpeg4","value":178},{"name":"av1","value":109},{"name":"vp9","value":13},{"name":"vc1","value":31},{"name":"mpeg2video","value":69}],[{"name":"mkv","value":19030},{"name":"mp4","value":2331},{"name":"avi","value":178},{"name":"webm","value":12},{"name":"m4v","value":2},{"name":"m2ts","value":1}],[{"name":"1080p","value":16525},{"name":"720p","value":1319},{"name":"480p","value":1255},{"name":"4KUHD","value":2032},{"name":"576p","value":422},{"name":"1440p","value":1}],[{"name":"eac3","value":1}],[{"name":"mkv","value":1}]],["I:\\","fIOAwdZDI",5396,1028,931.9100346509367,26014,[{"name":"Not required","value":1132},{"name":"Queued","value":3458},{"name":"Transcode error","value":37},{"name":"Transcode success","value":767},{"name":"Ignored","value":2}],[{"name":"Success","value":5396}],[{"name":"hevc","value":3925},{"name":"h264","value":1352},{"name":"av1","value":62},{"name":"vp9","value":13},{"name":"mpeg2video","value":26}],[{"name":"mkv","value":4514},{"name":"mp4","value":852},{"name":"webm","value":12}],[{"name":"1080p","value":4360},{"name":"4KUHD","value":661},{"name":"720p","value":251},{"name":"480p","value":105},{"name":"1440p","value":1}],[],[]],["D:\\","O8JiP_A_Q",8614,4374,1878.0320044411346,43629,[{"name":"Transcode success","value":3881},{"name":"Queued","value":238},{"name":"Not required","value":4361},{"name":"Transcode error","value":134}],[{"name":"Success","value":8523},{"name":"Queued","value":86},{"name":"Error","value":5}],[{"name":"hevc","value":8331},{"name":"h264","value":268},{"name":"mpeg4","value":2}],[{"name":"mkv","value":8199},{"name":"mp4","value":400},{"name":"avi","value":2}],[{"name":"720p","value":737},{"name":"1080p","value":6583},{"name":"480p","value":564},{"name":"576p","value":277},{"name":"4KUHD","value":440}],[],[]],["G:\\","_cAxLy-A5",4642,252,45.70042809471488,23843,[{"name":"Queued","value":4446},{"name":"Not required","value":172},{"name":"Transcode success","value":18},{"name":"Ignored","value":6}],[{"name":"Success","value":4642}],[{"name":"hevc","value":2265},{"name":"h264","value":2121},{"name":"mpeg4","value":164},{"name":"vc1","value":29},{"name":"mpeg2video","value":43},{"name":"av1","value":10}],[{"name":"mkv","value":3939},{"name":"avi","value":164},{"name":"mp4","value":527},{"name":"m4v","value":1},{"name":"m2ts","value":1}],[{"name":"1080p","value":3354},{"name":"4KUHD","value":448},{"name":"480p","value":532},{"name":"720p","value":153},{"name":"576p","value":145}],[],[]],["Q:\\","6q3XHHoC2dH",676,288,143.73108595609665,18653,[{"name":"Queued","value":626},{"name":"Not required","value":13},{"name":"Transcode success","value":37}],[{"name":"Success","value":674},{"name":"Error","value":1},{"name":"Queued","value":1}],[{"name":"h264","value":272},{"name":"hevc","value":372},{"name":"vc1","value":2},{"name":"mpeg4","value":1},{"name":"av1","value":1}],[{"name":"mkv","value":633},{"name":"mp4","value":14},{"name":"avi","value":1}],[{"name":"1080p","value":288},{"name":"4KUHD","value":360}],[{"name":"eac3","value":1}],[{"name":"mkv","value":1}]],["F:\\","l0IDf4JUZ",2305,205,42.469798483885825,15420,[{"name":"Queued","value":2236},{"name":"Not required","value":64},{"name":"Transcode success","value":5}],[{"name":"Success","value":2305}],[{"name":"hevc","value":1473},{"name":"h264","value":775},{"name":"av1","value":36},{"name":"mpeg4","value":11}],[{"name":"mkv","value":1745},{"name":"mp4","value":538},{"name":"avi","value":11},{"name":"m4v","value":1}],[{"name":"1080p","value":1940},{"name":"480p","value":54},{"name":"4KUHD","value":123},{"name":"720p","value":178}],[],[]]],"tdarrScore":"48.32","healthCheckScore":"99.57","processWarning":"","processWarningQueues":false,"table0Count":0,"table2Count":10450,"table3Count":171,"table4Count":86,"table5Count":21540,"table6Count":6,"table1Count":11004,"streamStats":{"duration":{"average":2357,"highest":11320,"total":5775811},"bit_rate":{"average":2300819,"highest":17861393,"total":5664613977},"nb_frames":{"average":58260,"highest":315424,"total":140930211}},"avgNumberOfStreamsInVideo":6.639092511830751,"languages":{"ara":{"count":1103},"may":{"count":552},"rum":{"count":806},"chi":{"count":1733},"heb":{"count":4},"ind":{"count":210},"nob":{"count":243},"est":{"count":106},"lav":{"count":97},"nor":{"count":363},"per":{"count":13},"srp":{"count":74},"jpn":{"count":6},"fre":{"count":24},"tgl":{"count":1}}}

**Comment by moostmartijn** [comment]
Here's mine as I have the same issue:
[3d18b2ea89fa5357cf6423b4e6985b46fa10195e.json](https://github.com/HaveAGitGat/Tdarr/files/13590430/3d18b2ea89fa5357cf6423b4e6985b46fa10195e.json)

**Comment by HaveAGitGat** [comment]
@ARRed98 you should copy the whole file and paste it, think you just selected the top part

**Comment by HaveAGitGat** [comment]
@moostmartijn not sure about yours. FYI files which are being processed won't show in the queues numbers. So you mentioned it wasn't updating while a file is being processed which is correct. The number will only update afterwards when the file is in the success or error table. So may be working as intended.

**Comment by ARRed98** [comment]
[3d18b2ea89fa5357cf6423b4e6985b46fa10195e.json](https://github.com/HaveAGitGat/Tdarr/files/13590508/3d18b2ea89fa5357cf6423b4e6985b46fa10195e.json)

**Comment by HaveAGitGat** [comment]
thanks

**Comment by HaveAGitGat** [comment]
Maybe one way to look into this would be to zip up your server data folder (can DM it to me on Reddit/Discord if preferred) and send it to me so I can recreated it my side (could do that after running some test files from https://home.tdarr.io/samples/ so don't need to use your own files)

Else I'll see if I can recreate the issue some other way.

**Comment by moostmartijn** [comment]
@HaveAGitGat , the numbers also didn't update after the transcode was finished. But... I think my issue is now solved.

What went wrong:
When I updated to the latest version I only changed `image: haveagitgat/tdarr:2.11.01` to `image: haveagitgat/tdarr:latest` without changing anything else. So I was using the same folders/mounts and files with the new version.

I now created a new app folder and only placed the backup there. I deployed the latest version of Tdarr into the new folder and restored the backup I created with `2.11.01`. 

The numbers do update now :)
